runtime/calamari/src/weights/pallet_utility.rs

```diff
@@ -57,49 +57,49 @@ pub trait WeightInfo {
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_utility::WeightInfo for SubstrateWeight<T> {
     fn batch(c: u32, ) -> Weight {
-        (26_456_000 as Weight)
+        (30_920_000 as Weight)
             // Standard Error: 5_000
-            .saturating_add((4_848_000 as Weight).saturating_mul(c as Weight))
+            .saturating_add((5_043_000 as Weight).saturating_mul(c as Weight))
     }
     fn as_derivative() -> Weight {
-        (6_056_000 as Weight)
+        (6_213_000 as Weight)
     }
     fn batch_all(c: u32, ) -> Weight {
-        (29_438_000 as Weight)
-            // Standard Error: 5_000
-            .saturating_add((5_027_000 as Weight).saturating_mul(c as Weight))
+        (38_194_000 as Weight)
+            // Standard Error: 7_000
+            .saturating_add((5_202_000 as Weight).saturating_mul(c as Weight))
     }
     fn dispatch_as() -> Weight {
-        (13_827_000 as Weight)
+        (13_985_000 as Weight)
     }
     fn force_batch(c: u32, ) -> Weight {
-        (32_871_000 as Weight)
+        (24_824_000 as Weight)
             // Standard Error: 5_000
-            .saturating_add((4_844_000 as Weight).saturating_mul(c as Weight))
+            .saturating_add((5_080_000 as Weight).saturating_mul(c as Weight))
     }
 }
 
 // For backwards compatibility and tests
 impl WeightInfo for () {
     fn batch(c: u32, ) -> Weight {
-        (26_456_000 as Weight)
+        (30_920_000 as Weight)
             // Standard Error: 5_000
-            .saturating_add((4_848_000 as Weight).saturating_mul(c as Weight))
+            .saturating_add((5_043_000 as Weight).saturating_mul(c as Weight))
     }
     fn as_derivative() -> Weight {
-        (6_056_000 as Weight)
+        (6_213_000 as Weight)
     }
     fn batch_all(c: u32, ) -> Weight {
-        (29_438_000 as Weight)
-            // Standard Error: 5_000
-            .saturating_add((5_027_000 as Weight).saturating_mul(c as Weight))
+        (38_194_000 as Weight)
```

Comment:
+ 1/3

Reply:
reproduced by second run, don't really see a reason but the change is small enough to ignore